README.md

```diff
@@ -226,6 +243,56 @@ This class implements `FileInfoInterface` and has additional methods for file op
 
 As you can see, additional methods help you to call API methods without direct API calls.
 
+## Addons operations
```

Comment:
And ClamAv? We don't have docs article yet, but still..

Reply:
Added